README.md

```diff
@@ -124,11 +124,65 @@ caching by way of expiry or ETags, I recommend using `faraday-http-cache` for
 your clients and setting `memoize` to `false`. Remote Record will eventually
 gain support for caching.
 
+### `remote_all`
+
+If you're able to fetch multiple records at once from the API, implement the
+`self.all` method on your remote record class. This should return an array of
+hashes that can be used to initialize a set of references.
+
+This can optionally take a block
+for authorization - note that it won't use the auth you've configured and that
```

Comment:
The authorization you configure on the client might be based on the instance of the reference, e.g. `config.authorization { |reference| reference.user github_auth_token }`. Since `all` is at the class level, it doesn't have access to this.